src/Metal/Defs.cs

```diff
@@ -348,7 +348,11 @@ public partial interface IMTLTexture {
 	}
 #endif // COREBUILD
 #if MONOMAC
+#if !NET
 	[NoiOS, NoTV, NoWatch, Mac (10,14)]
+#else
+	[SupportedOSPlatform ("macos10.14")]
```

Comment:
AFAIK not needed, am I wrong? is the min version.

Reply:
Good catch!